components/ide/jetbrains/backend-plugin/src/main/kotlin/io/gitpod/jetbrains/remote/GitpodTerminalService.kt

```diff
@@ -186,6 +187,8 @@ class GitpodTerminalService(session: ClientProjectSession) {
         exitTaskWhenTerminalWidgetGetsClosed(supervisorTerminal, shellTerminalWidget)
 
         listenForTaskTerminationAndTitleChanges(supervisorTerminal, shellTerminalWidget)
+
+        portForwardingService.monitorPortsOfPid(shellTerminalWidget, supervisorTerminal.pid)
```

Comment:
Reminder: This is correctly listening to the ports opened from the Process ID of Supervisor Terminals, but it doesn't make the message about ports forwarded appear in the terminal where we ran `gp tasks attach`.

Reply:
We've reported it to JetBrains [[1](https://gitpod.slack.com/archives/C02BRJLGPGF/p1666595916533549?thread_ts=1666385421.749759&cid=C02BRJLGPGF)].